### ISSUE TITLE
[no-unused-vars] on enums

### ISSUE THREAD

**Issue by rndmerle** [bradzacher/eslint-plugin-typescript#244]
**Repro**
.eslintrc.js
```JSON
{
  "rules": {
    "no-unused-vars": 1,
    "typescript/no-unused-vars": 1,
  }
}
```
tsconfig.json
```JSON
{
  "preserveConstEnums": true,
}
```
code :
![image](https://user-images.githubusercontent.com/79888/50108147-d06efb80-0234-11e9-8d5f-b87ec6240736.png)

**Expected Result**
no-unused-vars should not warn about individual values of enums

**Actual Result**
eslint warns about each enum's value (wheter it's actually used or not, but it's not the point)

**Additional Info**
I'm not really sure if it's an eslint-plugin-typescript issue or from eslint itself.

Thanks :)

**Versions**

| package                    | version |
| -------------------------- | ------- |
| `eslint` | `5.10.0` |
| `eslint-plugin-typescript` | `0.14.0` |
| `typescript-eslint-parser` | `21.0.2` |
| `typescript`               | `3.2.2` |

**Comment by bradzacher** [comment]
Will be fixed as part of 1.0.0

Duplicate of #235
Please use the search next time

**Comment by j-f1** [comment]
Duplicate of #235

**Comment by rndmerle** [comment]
Sorry I didn't try to search in closed issues :(
Thanks for the fast answer tho :)
